courses/free-courses-ml.md

```diff
@@ -1,33 +1,113 @@
 ### Index
 
 * [Android](#android)
+* [Bash/Shell](#bash)
```

Comment:
- Expand title of `bash/shell` section and fix its fragment id to get right cross linking.

```suggestion
* [Bash and Shell](#bash-and-shell)
```